src/export/InstanceExporter.ts

```diff
@@ -119,6 +119,9 @@ export class InstanceExporter {
     instanceDef.resourceType = instanceOfStructureDefinition.type; // ResourceType is determined by the StructureDefinition of the type
     instanceDef.instanceName = fshDefinition.id; // This is name of the instance in the FSH
 
+    // Add the profile we are making an instance of to meta.profile
+    // If we end up adding more metadata, we should wrap this in a setMetadata function
+    instanceDef.meta = { profile: [instanceOfStructureDefinition.url] };
```

Comment:
You should only do this if it is indeed an instance of a _profile_ (not a base _resource_).  In other words, if I do an instance of plain old `Patient`, then I should _not_ have the `meta.profile` set.

Reply:
This is my way of checking if something is a base FHIR resource:
```
if (
      instanceOfStructureDefinition.url !==
      `http://hl7.org/fhir/StructureDefinition/${instanceDef.resourceType}`
    ) 
```
But I want to double check before I push that there isn't some obvious `isABaseResource` flag that I am missing.

Reply:
I think the more formally correct way is to check that `instanceOfStructureDefinition.derivation === 'constraint'`.